docker/risingwave.toml

```diff
@@ -14,7 +14,7 @@ checkpoint_frequency = 10
 shared_buffer_capacity_mb = 4096
```

Comment:
(off-topic) I think we need to delete this file `docker/risingwave.toml` like #6814 did.